src/navigation/sim_planning/sim_planning/node_pursuit.py

```diff
@@ -25,144 +27,227 @@
 
 from transforms3d.euler import quat2euler
 
-# import required sub modules
-from driverless_common.point import Point
-
 # initialise logger
 LOGGER = logging.getLogger(__name__)
 
-# translate ROS image messages to OpenCV
-cv_bridge = CvBridge()
-
-# image display geometry
-SCALE = 20
-WIDTH = 20*SCALE # 10m either side
-HEIGHT = 20*SCALE # 20m forward
-ORIGIN = Point(0, 0)
-IMG_ORIGIN = Point(int(WIDTH/2), HEIGHT)
 
-# display colour constants
-Colour = Tuple[int, int, int]
-YELLOW_DISP_COLOUR: Colour = (0, 255, 255) # bgr - yellow
-BLUE_DISP_COLOUR: Colour = (255, 0, 0) # bgr - blue
-ORANGE_DISP_COLOUR: Colour = (0, 165, 255) # bgr - orange
+def getWheelPosition(pos_cog: List[float], heading: float) -> List[float]:
+    """
+    Gets the position of the steering axle from the car's center of gravity and heading
+    * param pos_cog: [x,y] coords of the car's center of gravity
+    * param heading: car's heading in rads
+    * return: [x,y] position of steering axle
+    """
+    #https://fs-driverless.github.io/Formula-Student-Driverless-Simulator/v2.1.0/vehicle_model/
+    cogToAxleDist = 0.4 #m
+    x_axle = pos_cog[0] + cos(heading)*cogToAxleDist
+    y_axle = pos_cog[1] + sin(heading)*cogToAxleDist
 
-LEFT_CONE_COLOUR = Cone.BLUE
-RIGHT_CONE_COLOUR = Cone.YELLOW
+    return [x_axle, y_axle]
 
 
-def robot_pt_to_img_pt(x: float, y: float) -> Point:
+def getRVWP(car_pos: List[float], path: np.ndarray, rvwpLookahead: int) -> List[float]:
```

Comment:
use snake_case for function names (we should really set up a style guide / style checker)